docs/UserGuide.md

```diff
@@ -128,19 +128,18 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting an elderly : `deleteElderly`
 
-Deletes the specified person from the address book.
+Deletes an elderly from NurseyBook.
 
-Format: `delete INDEX`
+Format: `/deleteElderly INDEX`
```

Comment:
If I'm not wrong, the command should not have the `/` in front?

Reply:
Good catch! I've also updated the command summary.